discord_bot.py

```diff
@@ -41,7 +41,7 @@ async def on_command_error(self, ctx, exc: Exception):
             message = "Command used is missing required argument"
             logging.warn(get_stack_trace_str(exc))
         else:
-            message = "Oh no! Something went wrong while running the command!"
+            message = "Oh no! Something went wrong while running the command! Please use **!bothelp** command for supported commands."

```

Comment:
Why was this added? This message is posted when something unexpected happens (i.e. bug in our program). Asking bothelp won't help lol

Reply:
explained with david - 4xx, 5xx error code differentiation